src/attestation/Attestation.ts

```diff
@@ -47,15 +54,21 @@ export default class Attestation implements IAttestation {
   public delegationId?: IAttestation['delegationId']
 
   public constructor(
-    requestForAttestation: IRequestForAttestation,
```

Comment:
Will implement helper from...() functions, that take care of checking and generating stuff, making the constructor have only one very streamlined behaviour, instead of taking care of all posibilities